server/websocket.go

```diff
@@ -1062,3 +1096,23 @@ func (p *Plugin) handleBotWSReconnect(connID, prevConnID, originalConnID, channe
 
 	return nil
 }
+
+func (p *Plugin) handleCaptionMessage(channelID, captionFromSessionID, captionFromUserID, text string, newAudioLenMs float64) {
+	// TODO: broadcast to participants only, https://github.com/mattermost/mattermost-plugin-calls/pull/609
+	p.publishWebSocketEvent(clientMessageTypeCaption, map[string]interface{}{
+		"user_id":    captionFromUserID,
+		"session_id": captionFromSessionID,
+		"text":       text,
+	}, &model.WebsocketBroadcast{ChannelId: channelID, ReliableClusterSend: true})
+
+	p.metrics.ObserveLiveCaptionsAudioLen(newAudioLenMs)
+}
+
+func (p *Plugin) handleMetricMessage(metricName public.MetricName) {
+	switch metricName {
+	case public.MetricPressureReleased:
+		p.metrics.IncLiveCaptionsPressureReleased()
+	case public.MetricTranscriberBufFull:
```

Comment:
Can we prefix the types so that they match the methods? `MetricPressureReleased` sounds very generic if you don't have the context in mind. I know it becomes verbose but it's a tradeoff I'd take.

Reply:
Yep, sounds good. I'm not averse to long names (I usually think my names are too long and I try to cut them down).